Update tested load numbers (#42759)
keycloak/keycloak: docs/guides/high-availability/partials/tested-load.adoc

</@profile.ifCommunity>

* 100,000 users
<@profile.ifProduct>
* 300 requests per second
</@profile.ifProduct>
<@profile.ifCommunity>

> * 1000 logins per second with 20000 refreshes per second.

Reviewer: ```suggestion
* 1000 logins per second with 20000 token refreshes per second.
```

> * 6 Pods

Reviewer: ```suggestion
* 6 Pods single-cluster setup across 3 availability zones in one region
```

> ** Each Pod with limits: 40vCPU and 8GB Memory

Reviewer: ```suggestion
** Each Pod with limits of 40 vCPU and 8 GB memory
```

> * Amazon Aurora PostgreSQL database

Reviewer: ```suggestion
* Amazon Aurora PostgreSQL multi-az database deployed in two availability zones
```

> ** Instance type: `db.r8g.16xlarge`

Reviewer: ```suggestion
** Instance type: `db.r8g.16xlarge` for both reader and writer instance
```
